.prettierignore

```diff
@@ -19,6 +19,7 @@
 
 /projects/js-toolkit/**/generators
 /projects/js-toolkit/**/lib
+/projects/js-toolkit/qa/**
```

Comment:
Why should we ignore prettier in these files? Does it cause issue?

Reply:
Because it is usually a project generated for customers (with our generator/@liferay-cli), following other coding standards (for example it may suit React or Angular standards), so it doesn't make too much sense to lint them according to our rules.

It's not that we cannot add them, but it is not worth the effort and may give us problems in the future. 

In fact, I've been talking with @magjed4289 about automating this kind of tests in the future, so we may need to move them out of here. It really depends on how we design the QA architecture for the future.